lib/variations.js

```diff
@@ -4,16 +4,16 @@ const simplePrefs = require('sdk/simple-prefs');
 let Advise = null;
 
 const variations = {
-  'passive-panel'() {
+  'regular-sync-offer'() {
     Advise = new Advisor();
     Advise.start();
   },
-  'auto-open-panel'() {
-    simplePrefs.prefs.autoOpenPanel = true;
+  'new-features-sync-offer'() {
```

Comment:
So one thing I don't know about how variations work: Is this code run every time the add-on starts up, and it consistently chooses the same branch per-user? If so, why not pass the branch name or `newFeaturesSyncOffer` directly into the advisor class instead of setting a pref? If not, then it makes more sense that you need to persist the value.